owlbot.py

```diff
@@ -15,5 +15,11 @@
 """This script is used to synthesize generated parts of this library."""
 
 import synthtool.languages.node as node
+from synthtool import shell
+
+# Regenerate Discovery types.
+shell.run(('npm','install'))
```

Comment:
I think we want these commands after main runs, or else you risk losing your changes with other postprocessing.

Also, if you wanted to save yourself a little processing time... technically you're just running a node script, you don't actually need to run npm install, you can just call the script directly on the filesystem.

https://github.com/googleapis/nodejs-bigquery/blob/d327e17716a6fc534fd411f3bd7e27e8d0c7f7d5/package.json#L46

Very minor nit though.

Reply:
oh, that's a good point. I added `npm install` so it can install the devDependencies that is used on the `gen-types.js` script ( like `discovery-tsd`) but I think the owlbot docker image already runs `npm install` for dev dependencies right ?

Reply:
yes it does!